### PR TITLE
Simplifies Argilla guide

### PR DESCRIPTION
**Pull request by dvsrepo**

### CODE REVIEW

**Review by osanseviero — APPROVED** [review]
Thanks a lot!! :fire:

File: docs/hub/spaces-sdks-docker-argilla.md [hunk 1]
```diff
@@ -10,22 +10,31 @@ In the next sections, you'll learn to deploy your own Argilla app and use it for
...
+
+<Tip>
+If you want to customize the title, emojis, and colors of your space, go to "Files and Versions" and edit the metadata of your README.md file.
+</Tip>
 
 <div class="flex justify-center">
 <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-argilla-duplicate-space.png"/>
 </div>
 
 Once you have duplicated the space, you'll see the `Building` status and once it becomes `Running` your space is ready to go. If you don't see the Argilla login UI refresh the page.
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-argilla.md:31]: Maybe we should change the wording here as it's not a proper duplication setup

File: docs/hub/spaces-sdks-docker-argilla.md [hunk 2]
```diff
@@ -10,22 +10,31 @@ In the next sections, you'll learn to deploy your own Argilla app and use it for
...
+</a>
+
+You need to define the **Owner** (your personal account or an organization), a **Space name**, and the **Visibility**. To interact with the Argilla app with Python, you need to setup the visibility to `Public`.
+
+<Tip>
+If you want to customize the title, emojis, and colors of your space, go to "Files and Versions" and edit the metadata of your README.md file.
+</Tip>
 
 <div class="flex justify-center">
 <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-argilla-duplicate-space.png"/>
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-argilla.md:28]: Maybe here we should show a screenshot of the Spaces creation page

**Review by osanseviero — COMMENTED** [review]
Two minor suggestions :)

File: docs/hub/spaces-sdks-docker-argilla.md (hunk 1 above)
Comment (dvsrepo) [reply docs/hub/spaces-sdks-docker-argilla.md]: done!

File: docs/hub/spaces-sdks-docker-argilla.md (hunk 2 above)
Comment (dvsrepo) [reply docs/hub/spaces-sdks-docker-argilla.md:35]: I've uploaded a new image and changed the url, I guess once the PR is merged here we're fine: https://huggingface.co/datasets/huggingface/documentation-images/discussions/42

Comment (osanseviero) [reply docs/hub/spaces-sdks-docker-argilla.md:35]: That PR Is merged :) the url here is still the same though

Comment (dvsrepo) [reply docs/hub/spaces-sdks-docker-argilla.md]: Now it's changed, sorry!

**Pull request merged**